DAL/Data Transfer Objects/UsersDTO.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace DAL.Data_Transfer_Objects
 {
-    class UsersDTO
+    public class UsersDTO
```

Comment:
Volgens de PR is Users en UserDetails ongebruikt, waarom blijven ze dan bestaan.

Reply:
Removed